FileIO/GMSHInterface.cpp

```diff
@@ -235,6 +236,9 @@ MeshLib::Mesh* GMSHInterface::readGMSHMesh(std::string const& fname)
 		material_ids.insert(material_ids.end(), materials.cbegin(),
 			materials.cend());
 	}
+
+	MeshLib::ElementValueModification::condense(*mesh);
```

Comment:
In Gmsh, material IDs are not necessary to start from zero and be continuous. e.g. it can be 1, 3, 6. but in ogs we need to change it to 0, 1, 2